html-css/week1/styles.css

```diff
@@ -0,0 +1,668 @@
+html,
```

Comment:
you have 2 html, remove one of them

Reply:
I need two because one is for the common properties with "body" tag and the other is only specific to "html" tag.